src/rpc/server_util.h

```diff
@@ -31,6 +34,7 @@ ChainstateManager& EnsureAnyChainman(const std::any& context);
 CBlockPolicyEstimator& EnsureFeeEstimator(const node::NodeContext& node);
 CBlockPolicyEstimator& EnsureAnyFeeEstimator(const std::any& context);
 CConnman& EnsureConnman(const node::NodeContext& node);
+interfaces::Miner& EnsureMiner(const node::NodeContext& node);
```

Comment:
@ryanofsky I'm not sure what the design philophy is behind `Ensure` and `EnsureAny`. The other methods here return more low level objects.

Perhaps related to what you said here:

> > I think of "Context" as being a passive container for state instead of an object you would call methods on. The top level interfaces like `interfaces::Chain` `interface::Node` `interfaces::Wallet` are actually stateless themselves, and only point to state stored in other places. (The idea is to let interface instances be created and destroyed freely, since with multiprocess support new processes can connect and create interface pointers that get destroyed on disconnection, and the state of the node shouldn't be affected too much from connections and disconnections. Anyway, the mining interface could follow this pattern.)